tembo-cli/src/cmd/apply.rs

```diff
@@ -823,10 +834,17 @@ fn get_postgres_config(
         }
     }
 
-    let config = shared_preload_libraries
+    let final_lodable_libs = Runtime::new().unwrap().block_on(get_loadable_libraries(
+        shared_preload_libraries.clone(),
+        extensions,
+    ));
+
+    let config = final_lodable_libs
+        .unwrap()
```

Comment:
I’m not sure is this unwrap definitely safe or does it need error handling?

Reply:
Thanks. Added error handling.